kdaglas/theStore-react
# #164530204 Enable user to fetch products

#### What does this Pr do?
Enable users to view all the products

#### Description of the task to be implemented?
- add the fetch all components
- add the fetchall reducer,
- create the fetchall view
- add a font to be used
- add the actions

#### How can this be manually tested?
- Fetch and check out this branch by running `git fetch origin ft-view-all-products-164530204 && git checkout ft-view-all-products-164530204`
- Install project dependencies and start the app by running `yarn install && yarn start`

#### Related Pivotal Tracker Story
[#164530204](https://www.pivotaltracker.com/story/show/164530204)
